server/internal/pdf_generator.go

```diff
@@ -0,0 +1,458 @@
+package internal
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/codescalers/cloud4students/models"
+	"github.com/pkg/errors"
+	"github.com/signintech/gopdf"
+)
+
+const (
+	greyColor     uint8 = 80
+	darkGreyColor uint8 = 60
+
+	startX float64 = 25
+	startY float64 = 30
+
+	logoPath       = "internal/img/codescalers.png"
```

Comment:
can we make this name more generic so it can be replaced e.g with threefold logo or any other company?